contributors/design-proposals/architecture/architecture.md

```diff
@@ -1,7 +1,7 @@
 # Kubernetes Design and Architecture
 
 A much more detailed and updated [Architectural
-Roadmap](../../devel/architectural-roadmap.md) is also available.
+Roadmap](architectural-roadmap.md) is also available.
```

Comment:
Can you change the link to ` /contributors/design-proposals/architecture/architectural-roadmap.md`?

Reply:
Done